src/server/gc_worker/gc_worker.rs

```diff
@@ -935,8 +945,13 @@ where
     ) -> Result<()> {
         GC_COMMAND_COUNTER_VEC_STATIC.unsafe_destroy_range.inc();
         self.check_is_busy(callback).map_or(Ok(()), |callback| {
```

Comment:
can we remove `check is busy` and use scheduler capacity instead? seems `GcTask::GcKeys` can bypass it. Mix-use may still cause potential problems in the future.

Reply:
Makes sense. I totally remove `check_is_busy`.

Reply:
@cfzjywxk @youjiali1995 Wanna take another look for any other concerns?

Reply:
None